.github/workflows/rust-test.yml

```diff
@@ -83,9 +83,6 @@ jobs:
           - toolchain: ${{ inputs.nightly_toolchain }}
             type: miri
       fail-fast: ${{ inputs.fail-fast == 'true' }}
-    container:
-      image: ghcr.io/foresightminingsoftwarecorporation/rust-test:${{ matrix.toolchain }}
-      options: --user root
```

Comment:
This change broke miri tests. The runner needs to use the toolchain specified in the matrix.